transferbot.sh

```diff
@@ -36,6 +36,24 @@ cat <<CREDS
 CREDS
 } > $(wb config path)
 
-wb data $@ --instance "$source_wiki_origin" |\
+read_entities() {
+  # entities are passed as either
+  # "Q42" which fetches the latest revision
+  # "Q42@123" which fetches revision 123
+  for entity in $@; do
+    case $entity in
```

Comment:
looks good and functional but I had to remind myself exactly how this case and pattern matching stuff works in bash

Reply:
Another option would be comparing the output of `cut -d "@" -f 1` and `cut -d "@" -f 2`. If they are the same, there is no `@` present. Would you find this easier to digest as a reader?